qubekit/workflow/workflow.py

```diff
@@ -30,6 +31,10 @@ class WorkFlow(SchemaBase):
         LocalResource(),
         description="The local resource options for the workflow like total memory and cores available.",
     )
+    fragmentation: WBOFragmenter = Field(
```

Comment:
I think we should make this optional now so we can recreate old workflows which did not use fragmentation.